cdap-metadata-spi/src/main/java/io/cdap/cdap/spi/metadata/ChangeRequest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2025 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.spi.metadata;
+
+/**
+ * The Change Request for ES/Spanner Changes.
```

Comment:
SPI module should not mention anything about the implementation.

In the future if a new extension is added, this comment will become outdated.